etc/kayobe/controllers.yml

```diff
@@ -134,23 +134,106 @@
 
 # A firewalld zone to set as the default. Default is unset, in which case the
 # default zone will not be changed.
-#controller_firewalld_default_zone:
+# FIXME: Try setting to drop
+controller_firewalld_default_zone: trusted
 
 # A list of firewall rules to apply. Each item is a dict containing arguments
 # to pass to the firewalld module. Arguments are omitted if not provided, with
 # the following exceptions:
 # - offline: true
 # - permanent: true
 # - state: enabled
-#controller_firewalld_rules:
+controller_firewalld_rules: "{{ controller_firewalld_rules_default | selectattr('enabled', 'true') | map(attribute='rules') | flatten | selectattr('network', 'in', network_interfaces) | selectattr('zone') | union(controller_firewalld_rules_extra) | unique | select }}"
+controller_firewalld_rules_extra: []
+controller_firewalld_rules_default:
+  # Common
+  - rules:
+      - service: ssh
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: dhcp
+        zone: "{{ provision_wl_net_name | net_zone }}"
+        network: "{{ provision_wl_net_name }}"
+        state: enabled
+      - service: ntp
+        zone: "{{ provision_wl_net_name | net_zone }}"
+        network: "{{ provision_wl_net_name }}"
+        state: enabled
+      - service: tftp
+        zone: "{{ provision_wl_net_name | net_zone }}"
+        network: "{{ provision_wl_net_name }}"
+        state: enabled
+      - port: 8089/tcp
+        zone: "{{ provision_wl_net_name | net_zone }}"
+        network: "{{ provision_wl_net_name }}"
+        state: enabled
+      - service: cockpit
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: disabled
+      - service: dhcpv6-client
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: disabled
+      - service: ssh
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: disabled
+    enabled: true
+  # Designate
+  - rules:
+      - port: 53/tcp
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: enabled
+      - port: 53/udp
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: enabled
+      - port: 5354/tcp
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: enabled
+      - port: 5354/udp
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: enabled
+    enabled: "{{ kolla_enable_designate | bool }}"
+  # GENEVE
+  - rules:
+      - port: 6081/udp
+        zone: "{{ tunnel_net_name | net_zone }}"
+        network: "{{ tunnel_net_name }}"
+        state: enabled
+    enabled: "{{ ('geneve' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types)) | bool }}"
```

Comment:
Strictly only necessary on network hosts - it's possible to make these separate. Could check the host is in the 'network' group?

Reply:
Done